drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -1,25 +1,57 @@
 
 #include <iostream>
-#include "Quadrotor.h"
-#include "drake/systems/Simulation.h"
-#include "drake/systems/plants/BotVisualizer.h"
+#include "BotVisualizer.h"
```

Comment:
no.  you should use the drake/systems/plants/BotVisualizer.h syntax.  (and remove the include_directory calls in CMakeLists.txt).  it's the new way.